src/main/java/com/spotify/autoscaler/db/BigtableCluster.java

```diff
@@ -66,7 +65,7 @@ default String clusterName() {
   }
 
   default int effectiveMinNodes() {
-    return Math.min(minNodes() + loadDelta(), maxNodes());
+    return Math.max(minNodes(), minNodesOverride());
```

Comment:
Since the other change is reverted, this should change to:

`Math.min(Math.max(minNodes(), minNodesOverride()), maxNodes())`

Reply:
Isn't that enforced somewhere else? Let me check..